test/extended/apibinding/apibinding_utils.go

```diff
@@ -0,0 +1,98 @@
+package apibinding
+
+import (
+	"fmt"
+
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+// APIBinding struct defination
```

Comment:
Typo of defination here. And another place.

Reply:
Addressed.